seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -236,7 +236,7 @@
         "filter is a useful method that can filter out values that don't match a certain criteria",
         "Let's remove all the values greater than five",
         "<code>array = array.filter(function(val) {</code>",
-        "<code>&thinsp;&thinsp;return val<4;</code>",
+        "<code>&thinsp;&thinsp;return val <= 5;</code>",
```

Comment:
@natac13 never do two things in a PR, specific for a particular issue. In this case **Attempt to fix poor description Where Art Thou**.

But you've updated the description of Where art thou bonfire as well as modified description in OOPs and Functional programming challenges in a single PR.

If that was by accident, you can delete the remote branch only and interactively rebase the local branch to only include the **Where art thou** bonfire description changes.

Else, if it was deliberate, you can rename this PR to something more generic such as **Update Descriptions**.